### ISSUE TITLE
July team meeting

### ISSUE THREAD

**Issue by betatim** [jupyterhub/team-compass#44]
This issue is to schedule and coordinate the July team meeting. We tried to schedule a June meeting in #43 but decided to postpone to this date.

We meet 19th July at 6pm Zurich ([in your timezone](http://arewemeetingyet.com/Zurich/2018-07-19/18:00/JupyterHub%20team)). Thursday as usual. 

* video link: https://calpoly.zoom.us/my/jupyter
* [ ] agenda link: XXX (carry over items from #43 and start hackpad)
* [ ] announce on: gitter, jupyter@googlegroups.com, jupyter-education@googlegroups.com, binderhub-dev@googlegroups.com

**Comment by choldgraf** [comment]
I added a hackmd link and copied over the items from the last issue w/ names attached to them - please re-organize as folks wish!

**Comment by jzf2101** [comment]
Slight issue - JLab is meeting at the same time. We're going to have a channel collision.  Correct @ellisonbg @jasongrout?

**Comment by jasongrout** [comment]
No, JupyterLab meets 14 hours from now, not 40 (i.e., Wednesday, not Thursday).
